switchStatement.js

```diff
@@ -18,7 +18,50 @@ Steps:
+  Horoscope: It's a good day to focus on your financial matters, Taurus. Take a look at your budget and make any necessary adjustments. Avoid impulsive spending and save for the future.`);
+break
+case "Gemini" :console.log(`(May 21 - June 20):
+  Horoscope: Your social calendar is likely to be full today, Gemini. You're in the mood for socializing and networking. Make new connections and engage in stimulating conversations. Keep an open mind.`)
+  ;
+  break
+  case "Cancer" :console.log(`(June 21 - July 22):
+    Horoscope: It's time to take care of yourself, Cancer. Pay attention to your emotional well-being and prioritize self-care. Take a break from your usual routine and indulge in some self-nurturing activities.`);
+    break
+  case "Leo" : console.log(`(July 23 - August 22):
+    Horoscope: Your creativity is on fire today, Leo. Express yourself through your artistic pursuits and let your inner child come out to play. Your confidence and charisma will attract attention.`);
+    break
+    case "Virgo" : console.log(`(August 23 - September 22):
+      Horoscope: It's a good day for organizing and decluttering, Virgo. Clean up your physical space and tidy up your thoughts. Focus on practical matters and set achievable goals.`);
+    break   
+  case "Libra" :console.log(` (September 23 - October 22):
+    Horoscope: Your diplomacy and charm are in full force today, Libra. Use your skills to resolve conflicts and bring harmony to your relationships. Seek balance and fairness in all your interactions.
+ `);
+ break
+ case "Scorpio" :console.log(`(October 23 - November 21):
+  Horoscope: Your intuition is heightened today, Scorpio. Trust your instincts and delve into your emotions. Explore your subconscious mind and gain insights into your inner workings.`);
+  break
+  case "Sagittarius" : console.log(`(November 22 - December 21):
+    Horoscope: You're in the mood for adventure, Sagittarius. Plan a trip or explore new possibilities. Your optimism and enthusiasm will inspire others, and you may learn something new.`
+ );
+ break
+ case "Capricorn":console.log(`(December 22 - January 19):
+  Horoscope: It's time to focus on your career goals, Capricorn. Set clear objectives and work diligently towards achieving them. Your hard work and determination will pay off in the long run.
+`);
+break 
+case "Aquarius" :console.log(`(January 20 - February 18):
+  Horoscope: Your humanitarian side is in the spotlight today, Aquarius. Engage in activities that promote social causes and make a positive impact. Connect with like-minded individuals and share your innovative ideas.
+`);
+break
+case "Pisces":`(February 19 - March 20):
+  Horoscope: Your sensitivity and intuition are heightened today, Pisces. Pay attention to your dreams and emotions, as they may hold important messages for you. Take some time for self-reflection and introspection.
+`
+default:console.log("Sorry, we do not have a horoscope for that sign."); }
```

Comment:
Please organize your code better next time:
- Start the statement of each case in the switch case statement on the next line below the case keyword, not next to it.
- End the curly bracket below all the switch statements.